src/instruments/src/Common/popup.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-undef */
```

Comment:
**suggestion:** move those files to `src/shared/`, as they might be used by non-instrument components such as the FMGC.